src/index.js

```diff
@@ -23,82 +27,108 @@ module.exports = new BaseKonnector(start)
+    })
+    await request({
+      uri: `${baseUrl}/prive/verifiersession/v1`,
+      method: 'POST',
+      headers: {},
+      body: {},
+      resolveWithFullResponse: true
+    })
 
-  if (listeAnneeRemuneration || listeAnneeRemunerationPension) {
-    return {
-      yearsPaie: listeAnneeRemuneration,
-      yearsPension: listeAnneeRemunerationPension
+    let neededCookies = resp.headers['set-cookie']
+    let test = neededCookies[0]
+
+    let cookieSplit = test.split('; ')
+    let cookieObject = {}
+    cookieSplit.forEach(function(value) {
+      let splitResult = value.split('=')
+      cookieObject[splitResult[0]] = splitResult[1]
+    })
+
+    let stringedAmount = doc.libelle3.replace(',', '.')
+    const splitDate = doc.dateEvenement.split('/')
+    const formatDay = splitDate[0]
+    const formatMonth = splitDate[1]
+    const formatYear = splitDate[2]
+    doc.dateEvenement = `${formatYear}-${formatMonth}-${formatDay}`
+
+    oneFile.push({
+      ...doc,
+      date: new Date(doc.dateEvenement),
+      vendor: VENDOR,
+      vendorRef: doc.evenementId,
+      amount: parseFloat(stringedAmount),
+      fileurl: `${baseUrl}/prive/telechargerremunerationpaie/v1?documentUuid=${doc.documentUuid}`,
+      filename: `${doc.libelle1.toLowerCase().replace(/ /g, '_')}.pdf`,
+      requestOptions: {
+        headers: {
+          'X-XSRF-TOKEN': `${cookieObject['XSRF-TOKEN']}`
+        }
+      },
+      fileAttributes: {
+        metadata: {
+          datetime: utils.formatDate(new Date()),
+          datetimeLabel: 'issueDate',
+          contentAuthor: 'ensap.gouv.fr',
+          carbonCopy: true,
+          qualification: Qualification.getByLabel('pay_sheet')
+        }
+      }
+    })
+    log('info', oneFile[0])
+    if (oneFile[0].service === 'remuneration') {
+      await this.saveBills(oneFile, fields, {
+        fileIdAttributes: ['vendorRef'],
+        linkBankOperations: false,
+        identifiers: ['Ensap'],
+        sourceAccountIdentifier: fields.identifiant
+      })
+    } else {
+      log('info', 'is no bill')
+      await this.saveFiles(oneFile, fields, {
+        fileIdAttributes: ['vendorRef'],
+        linkBankOperations: false,
+        identifiers: ['Ensap'],
+        sourceAccountIdentifier: fields.identifiant
+      })
     }
-  } else {
-    log('warn', 'could not find year of remuneration')
-    return {}
   }
 }
-
 async function authenticate(username, password) {
-  const resp = await request({
-    uri: `${baseUrl}/authentification`,
+  let resp = await request({
+    uri: `${baseUrl}/`,
     method: 'POST',
-    formData: { identifiant: username, secret: password }
+    Headers: { 'Content-Type': 'application/json' },
+    formData: { identifiant: username, secret: password },
+    resolveWithFullResponse: true
   })
-  if (resp.includes('Identifiant ou mot de passe erroné')) {
+  resp = resp.body
+
+  if (resp.message.includes('Identifiant ou mot de passe erroné')) {
```

Comment:
Are your sur you don't have better criterias to detect LOGIN_FAILED ? Something like http status or url redirection ?